src/lucky_record/criteria.cr

```diff
@@ -50,6 +50,10 @@ class LuckyRecord::Criteria(T, V)
     add_clause(LuckyRecord::Where::LessThanOrEqualTo.new(column, V::Lucky.to_db!(value)))
   end
 
+  def in(value)
+    add_clause(LuckyRecord::Where::In.new(column, V::Lucky.to_db!(value)))
```

Comment:
Rather than adding parsing to all the types, you could probably remove the new parse and to_db methods and instead map the values:

```
def in(values)
  values = values.map { |value| V::Lucky.to_db!(value) }
  add_clause(LuckyRecord::Where::In.new(column, values))
end
```

Then you'll have an array of values. I'm not if this will require changes elsewhere in LuckyRecord, but I hope not *fingers crossed*

Reply:
@mikeeus I tried this out locally and specs seemed to pass. Here's a commit on a test branch that you can check out. I *think* this should work: https://github.com/luckyframework/lucky_record/commit/4e03f4e23926e5b60d9072b44e323d9dfe7a5967

Reply:
Nice, this works very well